### ISSUE TITLE
Fuzzy search returns all objects on empty input

### ISSUE THREAD

**Issue by ziima** [pycountry/pycountry#206]
`pycountry.countries.search_fuzzy('')` returns all countries in somewhat random order. I would expect either an error or an empty result set. Not sure which one is better.

**Comment by hasanasim** [comment]
I agree, when `list(pycountry.countries)` provides the same data (albeit in a different order), this fuzzy search has no reason to be valid. 

I'm leaning towards the fix being a `LookupError("Empty string is not a valid term")` @zware thoughts on this? I can raise a PR tomorrow.
